osidb/tests/test_endpoints.py

```diff
@@ -1225,6 +1225,97 @@ def test_flaw_update_cve(
         assert body["embargoed"] == embargoed
         assert body["cve_id"] == new_cve_id
 
+    @pytest.mark.parametrize(
+        "embargoed,old_date,new_date,alerts",
+        [
+            (
+                False,
+                datetime(2011, 1, 1, tzinfo=timezone.utc),
+                datetime(2012, 1, 1, tzinfo=timezone.utc),
+                False,
+            ),
+            (
+                False,
+                datetime(2021, 1, 1, tzinfo=timezone.utc),
+                datetime(2022, 1, 1, tzinfo=timezone.utc),
+                True,
```

Comment:
why is this different from the above? maybe we can explain in the test method docstring each case?

Reply:
I will add some explanation. Basically the issue is that you are assigning a future public date to an already public flaw which is obviously not correct.